client/extensions/wp-super-cache/components/advanced.jsx

```diff
@@ -0,0 +1,570 @@
+		};
+
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="wp_max_time">
+					{ translate( 'Cache Timeout' ) }
+				</FormLabel>
+
+				<FormTextInput
+					id="wp_max_time"
+					onChange={ this.handleCacheTimeoutChange }
+					style={ style }
+					type="text"
+					value={ cacheTimeout } />
+				{ translate( 'seconds' ) }
+				<FormSettingExplanation>
+					{
+						translate(
+							'How long should cached pages remain fresh? Set to 0 to disable garbage collection. ' +
+							'A good starting point is 3600 seconds.'
+						)
+					}
+				</FormSettingExplanation>
+			</FormFieldset>
+		);
+	}
+
+	renderScheduler() {
+		const { translate } = this.props;
+		const { clock, timer, type } = this.state.scheduler;
+		const style = {
+			width: '100px',
+			marginLeft: '10px',
+			marginRight: '10px',
+		};
+
+		return (
+			<FormFieldset>
+				<FormLabel>
+					{ translate( 'Scheduler' ) }
+				</FormLabel>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'interval' === type }
+						onChange={ this.handleSchedulerTypeChange }
+						style= { { marginTop: '12px' } }
+						value="interval" />
+					<span>
+						{ translate( 'Timer' ) }
+						<FormTextInput
+							disabled={ 'interval' !== type }
+							onChange={ this.handleSchedulerTimerChange }
+							style={ style }
+							type="text"
+							value={ timer } />
+						{ translate( 'seconds' ) }
+					</span>
+				</FormLabel>
+				<FormSettingExplanation isIndented>
+					{ translate( 'Check for stale cached files every interval seconds.' ) }
+				</FormSettingExplanation>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'time' === type }
+						onChange={ this.handleSchedulerTypeChange }
+						style= { { marginTop: '12px' } }
+						value="time" />
+					<span>
+						{ translate( 'Clock' ) }
+						<FormTextInput
+							disabled={ 'time' !== type }
+							onChange={ this.handleSchedulerClockChange }
+							style={ style }
+							type="text"
+							value={ clock } />
+						{ translate( 'HH:MM' ) }
+					</span>
+				</FormLabel>
+				<FormSettingExplanation isIndented>
+					{ translate( 'Check for stale cached files at this time (UTC) or starting at this time ' +
+						'every interval below.' ) }
+				</FormSettingExplanation>
+			</FormFieldset>
+		);
+	}
+
+	renderInterval() {
+		const { translate } = this.props;
+		const { interval } = this.state.scheduler;
+
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="cache_scheduled_select">
+					{ translate( 'Interval' ) }
+				</FormLabel>
+
+				<FormSelect
+					id="cache_scheduled_select"
+					onChange={ this.handleSchedulerIntervalChange }
+					value={ interval }>
+					<option value="five_minutes_interval">{ translate( 'Once every five minutes' ) }</option>
+					<option value="jetpack_sync_interval">{ translate( 'Every 5 minutes' ) }</option>
+					<option value="minutes_10">{ translate( 'Every 10 minutes' ) }</option>
+					<option value="minutes_30">{ translate( 'Every 30 minutes' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( '30 Minutes' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).